ui/packages/catalog-ui-search/src/main/webapp/react-component/filter/filter-input/filter-location-input.js

```diff
@@ -79,6 +79,11 @@ class LocationInput extends React.Component {
     this.props.listenTo(this.locationModel, 'change:mode', () => {
       this.clearLocation()
     })
+    this.props.listenTo(this.locationModel, 'change:polygon', () => {
+      if (this.locationModel.get('mode') !== 'poly') {
+        wreqr.vent.trigger('search:polydisplay', this.locationModel)
```

Comment:
I think I'm just missing something but this seems odd. We only want to trigger the `polydisplay` if the mode is _not_ `poly`?

Reply:
Resolved offline. This fixes the aoi/target rendering issue, which has the modes `aoi` / `target`.